content/applications/inventory_and_mrp/manufacturing/management/split_merge.rst

```diff
@@ -0,0 +1,82 @@
+====================================
+Split and merge manufacturing orders
+====================================
+
+In Odoo *Manufacturing*, it is possible to create manufacturing orders for a single unit of an item
+or multiple units of the same item. In some cases, it may be necessary to split a manufacturing
+order that contains multiple units into two or more orders, or to merge two or more orders into a
+single order.
+
+.. important::
+   A manufacturing order can only contain one unit of a product or multiple units of a single
+   product that all use the same Bill of Materials (BoM). As a result, it is only possible to merge
+   manufacturing orders when every order contains the same product being manufactured with the same
+   BoM.
+
+Split manufacturing orders
+==========================
```

Comment:
```suggestion
Split
=====
```
Would it make sense to remove MO entirely from this heading? The title of the doc is quite self-explanatory: this doc is about the Split and merge features for MOs. So at a glance, would it be reasonable to assume the reader knows these sections are about the feature for manufacturing orders? Not to mention the first sentence of your sections makes it very clear: "To split a manufacturing order into multiple orders,..."

There's no need to take this suggestion, just a thought!